src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    const fillPeopleList = async () => {
+      setPeople(await getPeople());
+    };
+
+    try {
+      fillPeopleList();
+    } catch (error) {
+      throw new Error(`Unexpected error ${error}`);
```

Comment:
And you don't catch this error, create a stateful value, like `errorText` and use conditional rendering to show it